.gitignore

```diff
@@ -24,3 +24,6 @@ jetstream-*.tar
 
 # ASDF tool versions
 .tool-versions
+
+# tmp directory to test files
+tmp
```

Comment:
Could we us the ExUnit [tmp_dir](https://hexdocs.pm/ex_unit/main/ExUnit.Case.html#module-tmp-dir) here?

Reply:
👍 fixed